index.html

```diff
@@ -1,23 +1,37 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;0,800;1,400;1,600;1,700;1,800&family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="../whats-cookin/styles.css">
     <meta charset="UTF-8">
     <title>Whats Cookin</title>
   </head>
-
   <body>
-    <nav>
-      <button type="button" name="Home" id="home">Home</button>
-      <button type="button" name="favorites" id="favorites">Favorites</button>
-      <div class="header-section">
-        <p class="welcome-user">Welcome user</p>
-        <input type="text" class="search" name="search-field" value="What would you like to cook?">
-      </div>
+    <nav> 
```

Comment:
Resolved conflicts with html created by elements needed for CSS.  We're good.  Made a list of query selectors that will need to be updated.